### ISSUE TITLE
I cannot debug python code embedded in my C++ application.

### ISSUE THREAD

**Issue by vsfeedback** [microsoft/PTVS#7079]
_This issue has been moved from [a ticket on Developer Community](https://developercommunity.visualstudio.com/t/I-cannot-debug-python-code-embedded-in-m/10091030)._

---
I have a very simple C++ application which (either by pybind11 or Boost) makes a call to a function in a python script. I am unable to debug, or step into that python code.  
The program is working successfully, I have installed Python Tools for Visual Studio, I have also checked the debugger for python only is working.  
I have tried with two separate Visual Studio instances (one for C++ project and one for Python project) and starting the C++ project and then attaching to it, but the breakpoints in python code don’t get loaded. I have also tried with two projects inside the same Solution, but again it didn’t work (the breakpoints in python weren’t hit).  
I have read the [https://docs.microsoft.com/en-us/visualstudio/python/debugging-mixed-mode-c-cpp-python-in-visual-studio?view=vs-2022](https://docs.microsoft.com/en-us/visualstudio/python/debugging-mixed-mode-c-cpp-python-in-visual-studio?view=vs-2022) article, and went through it, but nevertheless I couldn’t get it to work.  
To summarize: I would like to start my C++ application and be able to debug the python code when the C++ application “calls” it.  
Is this possible to do, and how?  
(I am using Visual Studio 2019)

Here below is the screenshot of this simple C++ application I am using, which embeds the python interpreter an calls the python function.
![2022-07-08 18_06_20-PyBind - Microsoft Visual Studio (Administrator).png](https://aka.ms/dc/image?name=Bd552e34602f040f6a4f8942dc48e0cf1637928932079120048_2022-07-08_18_06_20-PyBind_-_Microsoft_Visual_Studio_(Administrator).png&tid=d552e34602f040f6a4f8942dc48e0cf1637928932079120048)

---
### Original Comments
(no comments)

---
### Original Solutions
(no solutions)

**Comment by int19h** [comment]
Are you using Python 3.10? If so, the mixed-mode debugger does not support it at the moment.

**Comment by VSladojevicFIS** [comment]
> Are you using Python 3.10? If so, the mixed-mode debugger does not support it at the moment.

Thanks, solved! I was using Python 3.9, but after your comment I tried with Python version 3.7 and it worked!
